src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(object, number) {
+      object.result += number;
+    },
+
+    subtract(object, number) {
+      object.result -= number;
+    },
+
+    multiply(object, number) {
+      object.result *= number;
+    },
+
+    divide(object, number) {
```

Comment:
Before performing division, there should be a check if 'number' is not equal to 0 to avoid division by zero error.